src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -102,7 +102,7 @@ export class NotebookMarkdownRenderer {
 			return '<img ' + attributes.join(' ') + '>';
 		};
 		renderer.link = (href: string, title: string, text: string): string => {
-			href = this.cleanUrl(!markdown.isTrusted, notebookFolder, href);
+			let returnedValue = this.cleanUrl(!markdown.isTrusted, notebookFolder, href);
 			if (href === null) {
```

Comment:
You never use returnedValue? Why aren't you setting href = this.cleanUrl(...)?

Reply:
I was testing something and forgot to change things back :(